drivers/tee/optee/Kconfig

```diff
@@ -12,3 +12,11 @@ config OPTEE_BENCHMARK
 	help
 	  This enables benchmarking feature in the OP-TEE Trusted
 	  Execution Environment (TEE) driver.
+
+config OPTEE_SHM_NUM_PRIV_PAGES
+	int "Private Shared Memory Pages"
+	default 1
+	depends on OPTEE
+	help
+	  This sets the number of private shared memory pages to be
+	  used by OP-TEE TEE driver.
```

Comment:
`s/OP-TEE TEE driver/OP-TEE driver/`

Reply:
But jbech-linaro "would just have written"
> ... by the OP-TEE TEE driver.

Reply:
Sorry missed that. I guess it's good as it is then.

Reply:
I will post it on LKML now.
Thanks all for reviewing.